docs/reference/latency.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Chainweb execution and latency
+description: "Benchmarks and calculations to help you understand expected execution times."
+id: latency
+sidebar_position: 5
+---
+
+# Chainweb execution and latency
+
+There are a number of factors that influence transaction execution, including network topology and the performance of individual nodes.
+This section provides benchmarks, estimations, and calculations to help you better understand the time it takes to execute transactions on Chainweb nodes under different circumstances.
+
+## Local execution
+
+In general, calls to the Chainweb `/local` endpoint return results almost immediately.
+Depending on how long it takes to execute the Pact code in a transaction, you can typically expect to see results from `/local` call within seconds, at most. 
+However, it's important to know that `/local` calls are executed on a single chain and served in a single-threaded, first-come-first-served basis.
+If a node is under contention—because it's catching up with state, busy processing multiple `/local` calls, or performing mempool checks—execution could take as long as needed for the request to be serviced in the queue.
+
+## Block production
+
+On average, producing a block takes 30 seconds.
+However, the average block production time represents the exponential distribution between events.
```

Comment:
I would delete this line in favor of putting the time distributions of each constituent process in the more specific sections.